tests/integration-tests/tests/scaling/test_scaling.py

```diff
@@ -121,6 +125,8 @@ def test_nodewatcher_terminates_failing_node(scheduler, region, pcluster_config_
     )
 
     assert_no_errors_in_logs(remote_command_executor, scheduler)
+    maintain_initial_size = cluster.config.get("cluster default", "maintain_initial_size")
```

Comment:
I mentioned this in a previous PR as well, don't like the general approach of how we are retrieving parameter values like `maintain_initial_size` from the config and using them in our test, because we never really enforce the coverage of the test. We should only do this if the parameter value is not related to test coverage.

Right now `test_scaling` has `maintain_initial_size = true` in the config and `test_sge` and `test_torque` has `maintain_initial_size = false` in the configs, so we are able to cover both cases today.
However, if we changed `maintain_initial_size = false` in `test_scaling` tomorrow, all the test will still pass, but we essentially have less test coverage because none of our test will cover the case of `maintain_initial_size = true`, and it would be difficult for us to notice this.

In this case it would be preferred to hard code things, or declare parameter values in the test explicitly, because then we would know exactly what we are testing for

Reply:
Oh, I see! Done